### PR TITLE
refactor: rename admin page variables in line with UI

### PR DESCRIPTION
**Pull request by mantariksh**
## Problem

The word "Builder" is used in the code to refer to both the "Create" page and the specific "Builder" tab under that page. This was confusing for me when reading the code for the first time and trying to match it to the UI.

## Solution

- Rename instances of "Builder" to "CreatePage" where relevant, to bring the naming in line with "SettingsPage"
- Rename instances of "BuilderDesign" to "BuilderAndDesign" when referring to components which include both the Builder and Design drawers
- Rename "ResponsesPage" to "ResultsPage" in line with UI

### CODE REVIEW

File: frontend/src/features/admin-form/create/CreatePageDrawerContext.tsx [hunk 1]
```diff
@@ -26,29 +29,29 @@ type BuilderDrawerContextProps = {
...
 
-const BuilderDrawerContext = createContext<
-  BuilderDrawerContextProps | undefined
+const CreatePageDrawerContext = createContext<
+  CreatePageDrawerContextProps | undefined
 >(undefined)
 
 /**
  * Provider component that makes drawer context object available to any
  * child component that calls `useBuilderDrawer()`.
```
Comment (karrui) [inline frontend/src/features/admin-form/create/CreatePageDrawerContext.tsx:38]: should we update this to `useCreatePageDrawer`?

File: frontend/src/features/admin-form/create/CreatePageDrawerContext.tsx [hunk 2]
```diff
@@ -26,29 +29,29 @@ type BuilderDrawerContextProps = {
...
 /**
  * Hook for components nested in ProvideAuth component to get the current auth object.
  */
-export const useBuilderDrawer = (): BuilderDrawerContextProps => {
-  const context = useContext(BuilderDrawerContext)
+export const useCreatePageDrawer = (): CreatePageDrawerContextProps => {
+  const context = useContext(CreatePageDrawerContext)
   if (!context) {
     throw new Error(
       `useBuilderDrawer must be used within a BuilderDrawerProvider component`,
```
Comment (karrui) [inline frontend/src/features/admin-form/create/CreatePageDrawerContext.tsx:57]: update this text too for consistency?

**Review by karrui — APPROVED** [review]
some minor changes for consistency, otherwise lgtm

File: frontend/src/features/admin-form/create/CreatePageDrawerContext.tsx (hunk 1 above)
Comment (mantariksh) [reply frontend/src/features/admin-form/create/CreatePageDrawerContext.tsx]: [804d3fc](https://github.com/opengovsg/FormSG/pull/3481/commits/804d3fc36b12deceaceade133d4697bba7e1c71f)

File: frontend/src/features/admin-form/create/CreatePageDrawerContext.tsx (hunk 2 above)
Comment (mantariksh) [reply frontend/src/features/admin-form/create/CreatePageDrawerContext.tsx]: [804d3fc](https://github.com/opengovsg/FormSG/pull/3481/commits/804d3fc36b12deceaceade133d4697bba7e1c71f)

**Pull request merged**